test/org/sagebionetworks/bridge/services/ScheduledActivityServiceMockTest.java

```diff
@@ -374,8 +381,9 @@ public void deleteActivitiesForSchedulePlanRejectsBadValue() {
     public void newActivitiesIncludedInSaveAndResultsV3() {
         List<ScheduledActivity> scheduled = createNewActivities("AAA", "BBB");
         List<ScheduledActivity> db = createStartedActivities("BBB");
+        Map<String, ScheduledActivity> dbMap = Maps.uniqueIndex(db, ScheduledActivity::getGuid);
         
-        List<ScheduledActivity> saves = merge(scheduled, db, V3_MERGE);
+        List<ScheduledActivity> saves = service.performMerge(scheduled, dbMap, V3_MERGE);
```

Comment:
Now that a lot of the merge logic is being handled outside of performMerge() (especially for V4), I don't believe this is sufficient. We need to move it up one level and test by calling getScheduledActivities().